frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSquashAction.java

```diff
@@ -119,15 +127,6 @@ private void doSquash(Project project,
       String branchName,
       boolean isSquashingCurrentBranch) {
 
-    @Data
-    // So that Interning Checker doesn't complain about enum comparison (by `equals` and not by `==`) in Lombok-generated `equals`
```

Comment:
Local classes are a specific cases of inner classes, there are no "static local classes" in Java